zhaquirks/tuya/ts0001_switch.py

```diff
@@ -0,0 +1,27 @@
+"""Tuya switch device."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.tuya import TuyaZBOnOffAttributeCluster
+
+
+class CustomMetering(Metering, CustomCluster):
+    """Tuya Valve Water consumed cluster."""
+
+    KILOWATT_HOURS = 0x0
+    ELECTRIC_METERING = 0x0
+
+    _CONSTANT_ATTRIBUTES = {
+        Metering.AttributeDefs.unit_of_measure.id: KILOWATT_HOURS,
+        Metering.AttributeDefs.metering_device_type.id: ELECTRIC_METERING,
+    }
+
+
+(
+    QuirkBuilder("_TZ3000_tgddllx4", "TS0001")
```

Comment:
Z2M has quite a few other device variants. Should I add them too, despite having no way to test them?
https://github.com/Koenkk/zigbee-herdsman-converters/blob/495ac226f381fa4a33eb12c809f9bd3f09e1216f/src/devices/tuya.ts#L3640

Reply:
That's been my philosophy, if they have the same converter on z2m, add them all on the quirk here.